neo/NeoSystem.cs

```diff
@@ -73,8 +73,7 @@ public void StartConsensus(Wallet wallet, Store consensus_store = null, bool ign
             Consensus.Tell(new ConsensusService.Start { IgnoreRecoveryLogs = ignoreRecoveryLogs }, Blockchain);
         }
 
-        public void StartNode(int port = 0, int wsPort = 0, int minDesiredConnections = Peer.DefaultMinDesiredConnections,
-            int maxConnections = Peer.DefaultMaxConnections)
+        public void StartNode(int port, int wsPort, int minDesiredConnections, int maxConnections)
```

Comment:
You are now passing `maxConnections` as a required param, won’t this always overwrite the default max connections from the confit? Same with `minDesiredConnections`?